x/evm/keeper/msg_server_test.go

```diff
@@ -134,3 +135,20 @@ func (suite *EvmKeeperTestSuite) TestUpdateParams() {
 		suite.Require().NoError(err)
 	}
 }
+
+func (suite *KeeperTestSuite) createContractMsgTx(nonce uint64, signer gethtypes.Signer, gasPrice *big.Int) (*types.MsgEthereumTx, error) {
```

Comment:
This is actually not something something new, I just moved it to this file because its not used in the old one anymore. This should be removed in a following PR that tackels the `msg_server` file. I was focusing on the `state_transition` file and removed the need for this function there.